test/v2/controllers/V2DeparturesControllerSpec.scala

```diff
@@ -1165,6 +1174,7 @@ class V2DeparturesControllerSpec
           mockDeparturesPersistenceService,
           mockRouterService,
           mockAuditService,
+          mockPushNotificationService,
           FakeMessageSizeActionProvider,
```

Comment:
Need to verify what gets returned when a PushNotificationError error is thrown

Reply:
Nothing does, it's currently fire and forget.